src/components/User/User.tsx

```diff
@@ -1,30 +1,38 @@
-import React,{ useState } from "react";
+import { useState } from "react";
 import css from "./User.module.css";
-import { Trash2 } from "react-feather";
+import { Trash2, Lock } from "react-feather";
+import UserType from "../../Interfaces/UserType";
 
 interface Props {
   id: number;
   pic: string;
   name: string;
   email: string;
+  status: string;
   hovered: (state: boolean) => void;
-  data: any;
-  findUser: any;
-
+  data: { id: number; pic: string; name: string; email: string }[];
+  findUser: (user: UserType) => void;
 }
 
-function User({ findUser, data, id, pic, name, email, hovered }: Props) {
-
+function User({
+  status,
+  findUser,
+  data,
+  id,
+  pic,
+  name,
+  email,
+  hovered,
+}: Props) {
   const [stateHovered, setStateHovered] = useState(false);
- 
 
   const mouseLeaveHandler = () => {
     setStateHovered(false);
     hovered(stateHovered);
     console.log(stateHovered);
   };
 
-  const hoverHandler = (id: any) => {
+  const hoverHandler = (id: number) => {
```

Comment:
Instead of iterating over users once again, move this function to parent component `UserList` and pass it as a prop to `User`. Use `obj` property received inside map loop to eliminate repeated iteration.